doc/usage_tutorial.md

```diff
@@ -10,7 +10,15 @@ To configurure parameters of the simulator:
 
 ## Running the simulator
 
-To run the simulator, simply use one of the launch files present in the [launch folder](../launch/):
+The simulator is inside a docker container, which by running it, the simulator will also be launched. To do this, run the following command on your terminal:
+
+```bash
+docker compose up
+```
+
+In case you want to do some work inside the container, by using the Dev Containers extension in VSCode the process becomes very simple, since the system is already setup for that.
```

Comment:
In case you want to do some work inside the container, by using the Dev Containers extension in VSCode the process becomes very simple, since there is a .devcontainer which contains the configuration for a development container for that purpose.